app/preview.py

```diff
@@ -62,6 +38,13 @@ def png_from_pdf(pdf_endpoint, page_number):
     }
 
 
+def get_logo_filename(dvla_org_id):
+    try:
+        return current_app.config['LOGO_FILENAMES'][dvla_org_id]
+    except KeyError:
+        abort(400)
+
```

Comment:
Write a unit test for this function.

Reply:
Test added in 3ff0394

Reply:
Can you add a TODO about the logo issue you mentioned yesterday?

Reply:
Added a story: https://www.pivotaltracker.com/story/show/153137328